plasma_cash/utils/merkle/sparse_merkle_tree.py

```diff
@@ -46,8 +47,30 @@ def create_tree(self, ordered_leaves, depth, default_nodes):
                     else:
                         next_level[index // 2] = sha3(default_nodes[level] + value)
                 prev_index = index
+            tree.append(tree_level)
```

Comment:
If you are changing the element inside I think in this case using this.tree might be better actually.

However, I think a better way here is to build the tree here and return the tree instead of root hash